src/App.tsx

```diff
@@ -6,25 +6,24 @@ import { AuthCheck } from "./AuthCheck";
 import { Homepage } from "./Homepage";
 import { Playlist } from "./Playlist";
 import { Playlists } from "./Playlists";
+import { ProtectedRoute } from "./ProtectedRoute";
 import { useAuth, ProvideAuth } from "./AuthContext";
 import { Test3 } from "./Test3";
 
 function App() {
   return (
     <ProvideAuth>
       <Routes>
-        <Route path="/login" element={<LoginButton />} />
```

Comment:
Similarly to playlists being wrapped in a `ProtectedRoute`, we should make another `LoggedOut` Route for all other components where if the user is logged in, we redirect to home.